tests/v2/unit/adapters/datumaro/adapter/test_visual_prompting_adapter.py

```diff
@@ -6,14 +6,16 @@
 
 from pathlib import Path
 from typing import TYPE_CHECKING, Union
+from datumaro.components.annotation import Mask, Polygon
+from datumaro.components.media import Image
 
 import numpy as np
 import pytest
 from otx.v2.adapters.datumaro.adapter.visual_prompting_dataset_adapter import (
     VisualPromptingDatasetAdapter,
 )
-from otx.v2.api.entities.image import Image
-from otx.v2.api.entities.shapes.polygon import Polygon
+# from otx.v2.api.entities.image import Image
+# from otx.v2.api.entities.shapes.polygon import Polygon
```

Comment:
Should we remove these lines?

Reply:
Thank you! I removed at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.